qutebrowser/utils/version.py

```diff
@@ -344,11 +344,10 @@ def _chromium_version():
     Also see https://www.chromium.org/developers/calendar
     and https://chromereleases.googleblog.com/
     """
-    if QWebEngineProfile is None:
+    if webenginesettings is None:
         # This should never happen
         return 'unavailable'
-    profile = QWebEngineProfile()
-    ua = profile.httpUserAgent()
+    ua = webenginesettings.default_user_agent
```

Comment:
I think that asserting that `ua` isn't none would be nice here, if the tests aren't relying on that.